### DISCUSSION TITLE
Different action_url for Thruk when using the CLI

### DISCUSSION THREAD

**Discussion by dan-m-joh** [sni/thruk#1100] [Q&A]
Hello Sven,

I have a question regarding the action_url in combination with Thruk.

Our action_url (for services) looks like this:
`action_url  https://<hostname>/grafana/dashboard/script/histou.js?host=$HOSTNAME$&service=$SERVICEDESC$&theme=light&annotations=true`

Is it possible to have/use a different url when we call `/usr/bin/thruk graph ...`?
The reason for asking is that we would like to set `...&annotations=false` when calling `/usr/bin/thruk graph ...`,
or do you have another suggestion on how to "remove" the annotations from the graph (we would like to keep then when using the "normal" Thruk-Interface)?

Regards,
Dan

**Comment by sni** [comment]
right now, there is no way to alter the url in any way. Maybe you could add a command line option to `thruk graph` to do so? Then just pass this option to get_perf_image() which needs a new option then as well. There are some already in
https://github.com/sni/Thruk/blob/master/lib/Thruk/Utils.pm#L1464-L1474
It gets called from here:
https://github.com/sni/Thruk/blob/master/lib/Thruk/Utils/CLI/Graph.pm#L134-L143
